client_integration_darwin_test.go

```diff
@@ -0,0 +1,41 @@
+package sftp
```

Comment:
why does this test only run on darwin ?

can ou rename it to client_integration_test.go and use 

// +build !windows

to rule out that os ?